modules/social_features/social_group/social_group.module

```diff
@@ -1325,6 +1336,12 @@ function social_group_block_access(Block $block, $operation, AccountInterface $a
     }
     return $access;
   }
+  // Show statistic block only when new style is enabled.
+  if ($operation == 'view' && $block->getPluginId() === 'group_statistic_block') {
+    if (theme_get_setting('style') !== 'sky') {
+      return AccessResult::forbidden();
+    }
+  }
```

Comment:
We control the block so we should probably just add this condition to the block.

Additionally, this creates a dependency of the block on the theme, which shouldn't happen. So we should find a way around this.

Reply:
@Kingdutch 
Added to the `blockAccess`.